lib/diplomat/plug/ensure_permitted.ex

```diff
@@ -29,18 +30,13 @@ defmodule Diplomat.Plug.EnsurePermitted do
 
   defp fetch_resource({nil, conn}, opts), do: handle_error(:unauthenticated, conn, opts)
   defp fetch_resource({actor, conn}, opts) do
-    resource = Diplomat.Plug.current_resource(conn)
+    resource = Plug.current_resource(conn)
     {actor, resource, conn, opts}
   end
 
+  defp permitted?(conn, _), do: conn
```

Comment:
I think you're right. It didn't break tests so I didn't catch this. Just was a bit more readable to my eyes when I moved it.